filesystem/local_test.go

```diff
@@ -70,28 +70,28 @@ func (s *LocalTestSuite) TestAllDirectories() {
 	s.True(s.local.Exists("AllDirectories/3/5/6/6.txt"))
 	files, err := s.local.AllDirectories("AllDirectories")
 	s.Nil(err)
-	if runtime.GOOS == "windows" {
+	if env.IsWindows() {
```

Comment:
Hi @kkumar-gcc You can use this method in your PR.

Reply:
Okay